client/src/Components/Login/index.jsx

```diff
@@ -1,11 +1,74 @@
 import React from 'react';
-import UnderConstruction from '../UnderConstruction';
-
-export default function Login() {
-  return (
-    <>
-      <h1>LOGIN</h1>
-      <UnderConstruction />
-    </>
-  );
+import './style.css';
+import { Form, Button, Container } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import signup from '../Signup';
+
+export default class Login extends React.Component {
+  state = {
+    username: '',
+    password: '',
+  };
+
+  handleClick = () => {
+    // make a requset to the back with method post and data{username , password}
+  };
+
+  handleChange = ({ target: { value, name } }) =>
+    this.setState({ [name]: value });
+
+  render() {
+    const { username, password } = this.state;
+
+    return (
+      <Container>
+        <Form className="content-login">
+          <span className="content-login__word-login">Login</span>
+          <Form.Group
+            controlId="formBasicUsername"
+            className="content-login__input "
+          >
+            <Form.Label>Username : </Form.Label>
+            <Form.Control
+              type="text"
+              name="username"
+              value={username}
+              placeholder="e.g: emily1234"
+              onChange={this.handleChange}
+              required
```

Comment:
look here: https://react-bootstrap.netlify.com/components/forms/#forms 
```Form.Control``` in react bootstrap doesn't take the ```required```
 so, I think you need to add validation for fields as required just
```suggestion
             
```